lib/formats/mgh.cpp

```diff
@@ -112,6 +122,7 @@ namespace MR
       for (const auto& tag : MGHO.tags) {
         out.write (reinterpret_cast<const char*> (&tag.id), sizeof(tag.id));
         out.write (reinterpret_cast<const char*> (&tag.size), sizeof(tag.size));
+        assert (size_t(ByteOrder::BE (tag.size)) == tag.content.size());
```

Comment:
That assert will fail routinely, because of the null termination on the `tag.content` string. For example, in the data sent through by @ansk:
```
...
mrinfo [lib/formats/mgh.cpp: 63]: tag.id = 33
mrinfo [lib/formats/mgh.cpp: 64]: tag.size = 1600
mrinfo [lib/formats/mgh.cpp: 65]: tag.content.size() = 185
mrinfo [lib/formats/mgh.cpp: 66]: tag.content = AutoAlign   1.000000   0.000000   0.000000   0.000000   0.000000   1.000000   0.000000   0.000000   0.000000   0.000000   1.000000   0.000000   0.000000   0.000000   0.000000   1.000000
...
```
Not sure whether this has anything to do with FreeSurfer not accepting these data, but that's how it worked before when it was confirmed as working. I'm guessing it'll be something else...